### PR TITLE
Added zh-cn localization.

### PR DESCRIPTION
**Pull request by ArsenalBastion4093**
Sorry for using "zh-cn" launguage code. However, the launguage code "zh" is blurry - "Which kind of Chinese, Simplified Chinese (zh-cn) or Traditional Chinese (zh-tw)?"

### CODE REVIEW

File: res/i18n/strings_zh-cn.i18n.json [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+        "appdsp": "APP + DS/P",
+        "appdspDescription": "只是每块发送垃圾行数与每块挖掘之和。",
+        "cheese": "垃圾行\n混乱指数",
+        "cheeseDescription": "(Cheese Index, 简称Cheese) 是玩家发出的垃圾行有多整齐/混乱的近似值。低数据代表整齐，高数据代表混乱。\n由 kerrmunism 发明",
+        "gbe": "垃圾行\n效率",
+        "gbeDescription": "(Garbage Efficity, 简称Gb Eff.) 测量玩家如何很好地利用他们收到的垃圾行。高数据代表更好或者他们更多地用TA的垃圾行，低数据代表TA大多将垃圾行送回奶酪层，或者很少清理垃圾行。\n由 Zepheniah 与 Dragonboy 发明。",
+        "nyaapp": "加权\nAPP",
+        "nyaappDescription": "(Weighted APP, 简称wAPP) 在本质上是在衡量您在保持高 APP 的同时发送奶酪的能力。\n由 Wertj 发明。",
+        "area": "面积",
+        "areaDescription": 如果排除 Cheese 和 vs/apm 部分，您的形状在图表上占据了多少空间",
```
Comment (dan63047) [inline res/i18n/strings_zh-cn.i18n.json:302]: You forgot to add "

**Review by dan63047 — CHANGES_REQUESTED** [review]
Need one small fix

File: res/i18n/strings_zh-cn.i18n.json (hunk 1 above)
Comment (ArsenalBastion4093) [reply res/i18n/strings_zh-cn.i18n.json]: Fixed.
Btw, thank you for accepting my translation!

**Review by dan63047 — APPROVED** [review]
Appreciate your help.

Please do note, that once redesign will be released, i may request you to update your localization

**Pull request merged**